playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -514,6 +514,9 @@ EDXAPP_INSTALL_PRIVATE_REQUIREMENTS: false
 #   - name: git+https://git.myproject.org/MyProject#egg=MyProject
 EDXAPP_EXTRA_REQUIREMENTS: []
 
+# Prevent arbitrary packages from being installed by the extra requirements
```

Comment:
```suggestion
# To prevent arbitrary packages from being installed by the extra requirements set 
#      EDXAPP_EXTRA_REQUIREMENTS_EXTRA_ARGS = '--no-deps'
```